stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/actions/httpclient/AbstractHTTPAction.java

```diff
@@ -369,8 +369,8 @@ private void serializeMultiparts(Context context, OutputStream messageOutputStre
 
     private BStruct getEntityStruct(Context context) {
         BStruct requestStruct = ((BStruct) context.getRefArgument(1));
-        return requestStruct.getNativeData(MESSAGE_ENTITY) != null ?
-                (BStruct) requestStruct.getNativeData(MESSAGE_ENTITY) : null;
+        return requestStruct.getRefField(REQUEST_ENTITY_INDEX) != null ?
+                               (BStruct) requestStruct.getRefField(REQUEST_ENTITY_INDEX) : null;
```

Comment:
Can't we just say, `return (BStruct) requestStruct.getRefField(REQUEST_ENTITY_INDEX);` here? I suppose we can cast `null` to any type.